src/components/discovery/DiscoverySearchForm.js

```diff
@@ -306,103 +315,136 @@ class DiscoverySearchForm extends Component {
 
         // longest title padded with marginRight
         return (
-            <Menu style={{display: "inline-block" }} onClick={this.addConditionFromPulldown}>
+            <Menu style={{ display: "inline-block" }} onClick={this.addConditionFromPulldown}>
                 <Menu.SubMenu title={<span>Subject</span>}>
-                {subjectOptions.map(o => <Menu.Item key={o.path}><DropdownOption option={o}/></Menu.Item>)}
+                    {subjectOptions.map((o) => (
+                        <Menu.Item key={o.path}>
+                            <DropdownOption option={o} />
+                        </Menu.Item>
+                    ))}
                 </Menu.SubMenu>
-                <Menu.SubMenu title={<span style={{marginRight: "10px"}}>Phenotypic Features </span>}>
-                {phenotypicFeaturesOptions.map(o => <Menu.Item key={o.path}><DropdownOption option={o}/></Menu.Item>)}
+                <Menu.SubMenu title={<span style={{ marginRight: "10px" }}>Phenotypic Features </span>}>
+                    {phenotypicFeaturesOptions.map((o) => (
```

Comment:
the mapped function is repeated ~6 times, probably worth factoring it out?

Reply:
this is one of several useless whitespace changes noted above. This code is ancient!

Reply:
oops